### ISSUE TITLE
viminfo with neovim

### ISSUE THREAD

**Issue by rosshadden** [clvv/fasd#91]
I was trying to get `fasd` working with neovim's equivalent of viminfo, which is [shada](https://neovim.io/doc/user/starting.html#shada-file).

```sh
_FASD_VIMINFO="$XDG_DATA_HOME/nvim/shada/main.shada"
```

Has anyone had any luck with this?  I'm guessing the shada file is not in the same format as viminfo, and thus the solution is that I need to create a new backend.  Just wanted to ask here in case someone else has already hit this.

**Comment by haifengkao** [comment]
I made something similar with neovim's python interface.

You can check it out
https://github.com/haifengkao/nfasd

**Comment by nhooyr** [comment]
this is all I do https://github.com/nhooyr/dotfiles/blob/2cb844075ad477fd0035c8b47dd1eb2e910b936f/.config/nvim/init.vim#L76-L85

I can even access it from within neovim with fzf, here https://github.com/nhooyr/dotfiles/blob/2cb844075ad477fd0035c8b47dd1eb2e910b936f/.config/nvim/init.vim#L277

**Comment by haifengkao** [comment]
brilliant!! 
nfasd has gone beyond the original goal. It now has folder completion, and better file ordering.
actually I am using it to replace fasd gradually. 
besides, it's written in Python (~_^)

**Comment by scarab5q** [comment]
Yeah I've run into the same issue, I'd like to write a back end for neovim shada file

**Comment by beardedsakimonkey** [comment]
here's a rough fasd backend for neovim's shada file. I have this in my `~/.fasdrc`:
```
shada() {
    ~/code/neovim/scripts/shadacat.py ~/.local/share/nvim/shada/main.shada | \
        sed -n "s/^.*'f': '\([^']*\)'.*$/\1/p" | \
        awk -v t="$(date +%s)" '{
            t -= 60
                print $0 "|1|" t
        }'
}
```
